main.py

```diff
@@ -0,0 +1,81 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from pydantic import BaseModel
+from flask import request
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///animals.db'
+db = SQLAlchemy(app)
+migrate = Migrate(app, db)
+
+class Animal(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(100))
+    breed = db.Column(db.String(100))
+    birth_date = db.Column(db.Date)
+    photo_url = db.Column(db.String(200))
+
+class AnimalFilter(BaseModel):
+    name: str
+
+@app.route('/health')
+def health_check():
+    return '', 200
+
+@app.route('/animals', methods=['GET'])
+def get_animals():
+    animals = Animal.query.all()
+    return jsonify([animal.__dict__ for animal in animals])
```

Comment:
You can do it in such way, but we used pydantic as additional level between sqlalchemy and response.
It give us advantages like validation and ability to use computed fields, etc.